packages/forma-36-react-components/src/components/Table/TableBody/TableBody.tsx

```diff
@@ -1,9 +1,14 @@
 import React, { Component } from 'react';
 
+import SkeletonTable from '../../Skeleton/SkeletonTable';
+
 export type TableBodyProps = {
   className?: string;
   style?: React.CSSProperties;
   testId?: string;
+  loading?: boolean;
+  numberOfRows?: number;
+  numberOfCells?: number;
```

Comment:
```suggestion
  loadingColumnCount?: number;
```

Reply:
Makes sense! Make it specific to loading and not the number of cells of the table

thanks!